### ISSUE TITLE
Crashing on Windows 10

### ISSUE THREAD

**Issue by RMichaelPickering** [MartinBriza/MediaWriter#67]
MediaWriter crashes almost immediately after running on Windows10 64bit laptop. Please find screen capture of error message 
[MediaWriter Error Win10.pdf](https://github.com/MartinBriza/MediaWriter/files/819556/MediaWriter.Error.Win10.pdf)
attached.

**Comment by MartinBriza** [comment]
Hello,
sorry but I cannot determine what went wrong in your case. Can you please be more specific about your hardware, your graphics card driver and if there is any special setting to your system? Especially user rights settings are important in this case.

**Comment by RMichaelPickering** [comment]
Hi Martin,

I am running a very new graphics/gaming laptop, the ASUS ROG Strix GL502VS.
It uses an Intel i7 quad core 6700HQ processor along with an Nvidia GTX
1070 graphics processor. I'm unaware of any special user rights settings
but am happy to have a look if you can tell me how.

Regards,

Mike


R. Michael Pickering CBIP
rmichaelpickering@gmail.com
Mobile: 321-337-2824 (Orlando)
Mobile: 416-721-1826 (Toronto)

**Comment by MartinBriza** [comment]
If you haven't set anything personally and it's not a corporate-owned (and managed) laptop, there's probably nothing special about your configuration. I'll check once more on my testing machines if there's anything similar to your behavior and will check back. If I don't find anything, I'll send you a special debug build that'll give us more information about the culprit of your problem.

**Comment by MartinBriza** [comment]
Hello, please try running this: https://mbriza.fedorapeople.org/FedoraMediaWriter-win32-4.0.8-57-g278efc9-debug.exe . Download will be available in about an hour (upload speeds are pretty bad and it's 375MB large).
It will put a FedoraMediaWriter.log file into your Documents folder. Please upload it here so we can see what is happening under the hood.

**Comment by MartinBriza** [comment]
Please try downloading this release: https://github.com/MartinBriza/MediaWriter/releases/tag/4.0.95 and check if it still has the problem you reported.

**Comment by kaleem4** [comment]
I Tried "Universal-USB-Installer-1.9.7.7" working Fine for me TESTED 
But Every time i tried FMW  Make me Disappointed really :( 
https://github.com/MartinBriza/MediaWriter/issues/21#issuecomment-285094132

**Comment by MartinBriza** [comment]
@kaleem4 I'm really sorry you're disappointed with how FMW behaves for you. Please, if want to help me solve your problems, open a new bug report with your FedoraMediaWriter.log created by FMW 4.0.95. I'll do my best to fix the bug for you.

**Comment by ckooiker** [comment]
release 4.0.95 works for me.

**Comment by kaleem4** [comment]
@MartinBriza Kindly Check Now how it would be solve ? Tried 4.0.95
```
D@11ms: Application constructed
D@11ms: Injecting QML context properties
D@15ms: WinDriveProvider construction
D@15ms: DriveManager construction
D@16ms: ReleaseManager construction
D@17ms: ReleaseManager Received a metadata json
D@17ms: ReleaseManager Adding "workstation" "25" "x86_64"
D@17ms: ReleaseManager Adding "workstation" "25" "i386"
D@18ms: ReleaseManager Adding "workstation" "25" "armhfp"
D@18ms: ReleaseManager Adding "server" "25" "x86_64"
D@18ms: ReleaseManager Adding "server" "25" "i386"
D@18ms: ReleaseManager Adding "server" "25" "armhfp"
D@18ms: ReleaseManager Adding "astronomy" "25" "x86_64"
D@19ms: ReleaseManager Adding "design suite" "25" "x86_64"
D@19ms: ReleaseManager Adding "games" "25" "x86_64"
D@19ms: ReleaseManager Adding "jam" "25" "x86_64"
D@19ms: ReleaseManager Adding "robotics" "25" "x86_64"
D@19ms: ReleaseManager Adding "scientific" "25" "x86_64"
D@19ms: ReleaseManager Adding "security" "25" "x86_64"
D@19ms: ReleaseManager Adding "astronomy" "25" "i386"
D@19ms: ReleaseManager Adding "design suite" "25" "i386"
D@19ms: ReleaseManager Adding "games" "25" "i386"
D@20ms: ReleaseManager Adding "jam" "25" "i386"
D@20ms: ReleaseManager Adding "robotics" "25" "i386"
D@20ms: ReleaseManager Adding "scientific" "25" "i386"
D@20ms: ReleaseManager Adding "security" "25" "i386"
D@20ms: ReleaseManager Adding "cinnamon" "25" "x86_64"
D@20ms: ReleaseManager Adding "kde" "25" "x86_64"
D@21ms: ReleaseManager Adding "lxde" "25" "x86_64"
D@21ms: ReleaseManager Adding "mate" "25" "x86_64"
D@21ms: ReleaseManager Adding "soas" "25" "x86_64"
D@21ms: ReleaseManager Adding "xfce" "25" "x86_64"
D@21ms: ReleaseManager Adding "cinnamon" "25" "i386"
D@21ms: ReleaseManager Adding "kde" "25" "i386"
D@21ms: ReleaseManager Adding "lxde" "25" "i386"
D@21ms: ReleaseManager Adding "mate" "25" "i386"
D@21ms: ReleaseManager Adding "soas" "25" "i386"
D@22ms: ReleaseManager Adding "xfce" "25" "i386"
D@22ms: ReleaseManager Adding "kde" "25" "armhfp"
D@22ms: ReleaseManager Adding "lxde" "25" "armhfp"
D@22ms: ReleaseManager Adding "mate" "25" "armhfp"
D@22ms: ReleaseManager Adding "soas" "25" "armhfp"
D@22ms: ReleaseManager Adding "xfce" "25" "armhfp"
D@22ms: ReleaseManager Adding "workstation" "24" "x86_64"
D@22ms: ReleaseManager Adding "workstation" "24" "i386"
D@22ms: ReleaseManager Adding "server" "24" "x86_64"
D@23ms: ReleaseManager Adding "kde" "24" "x86_64"
D@23ms: ReleaseManager Adding "kde" "24" "i386"
D@23ms: ReleaseManager Adding "xfce" "24" "x86_64"
D@23ms: ReleaseManager Adding "xfce" "24" "i386"
D@23ms: ReleaseManager Adding "lxde" "24" "x86_64"
D@23ms: ReleaseManager Adding "lxde" "24" "i386"
D@23ms: ReleaseManager Adding "mate" "24" "x86_64"
D@23ms: ReleaseManager Adding "mate" "24" "i386"
D@23ms: ReleaseManager Adding "cinnamon" "24" "x86_64"
D@24ms: ReleaseManager Adding "cinnamon" "24" "i386"
D@24ms: ReleaseManager Adding "soas" "24" "x86_64"
D@24ms: ReleaseManager Adding "soas" "24" "i386"
D@24ms: ReleaseManager Adding "astronomy" "24" "x86_64"
D@24ms: ReleaseManager Adding "astronomy" "24" "i386"
D@24ms: ReleaseManager Adding "design" "24" "x86_64"
D@24ms: ReleaseManager Adding "design" "24" "i386"
D@24ms: ReleaseManager Adding "games" "24" "x86_64"
D@24ms: ReleaseManager Adding "games" "24" "i386"
D@25ms: ReleaseManager Adding "robotics" "24" "x86_64"
D@25ms: ReleaseManager Adding "robotics" "24" "i386"
D@25ms: ReleaseManager Adding "scientific" "24" "x86_64"
D@25ms: ReleaseManager Adding "scientific" "24" "i386"
D@25ms: ReleaseManager Adding "security" "24" "x86_64"
D@25ms: ReleaseManager Adding "security" "24" "i386"
D@26ms: VersionChecker Asking for new FMW version information
D@347ms: Download Trying to download from a mirror: QUrl("https://getfedora.org/static/fmw-version.json")
D@347ms: Loading the QML source code
D@1375ms: Starting the application
D@1375ms: WinDriveProvider Looking for the drives for the first time
W@1375ms: Could not get disk extents for "\\\\.\\E:"
D@1375ms: WinDriveProvider Finished looking at drive letters
D@1375ms: WinDriveProvider "\\\\.\\PhysicalDrive0" is present
D@1375ms: WinDriveProvider "\\\\.\\PhysicalDrive0" IOCTL_STORAGE_QUERY_PROPERTY
D@1375ms: WinDriveProvider "\\\\.\\PhysicalDrive0" IOCTL_STORAGE_QUERY_PROPERTY with a bigger buffer
D@1375ms: WinDriveProvider "\\\\.\\PhysicalDrive0" detected: "" "ST500DM002-1BD142" , nonremovable notUSB
D@1376ms: WinDriveProvider "\\\\.\\PhysicalDrive1" is present
D@1376ms: WinDriveProvider "\\\\.\\PhysicalDrive1" IOCTL_STORAGE_QUERY_PROPERTY
D@1376ms: WinDriveProvider "\\\\.\\PhysicalDrive1" IOCTL_STORAGE_QUERY_PROPERTY with a bigger buffer
D@1376ms: WinDriveProvider "\\\\.\\PhysicalDrive1" detected: "SanDisk" "Cruzer Blade" , removable USB
D@1376ms: WinDriveProvider "\\\\.\\PhysicalDrive1" IOCTL_DISK_GET_DRIVE_GEOMETRY
D@1376ms: WinDriveProvider "\\\\.\\PhysicalDrive1" cleanup, adding to the list
D@1378ms: WinDriveProvider Finished looking for the drives for the first time
D@1379ms: Download Trying to download from a mirror: QUrl("https://getfedora.org/releases.json")
C@1505ms: === CRASH OCCURED ===
C@1505ms: An unhandled exception occurred:
C@1505ms: Code: -1073741819 - ACCESS VIOLATION
C@1505ms: Flags: 0
C@1505ms: Address 0x102095ed
C@1671ms: Backtrace:
C@1671ms: 	 4 :  0x0
C@1671ms: 	 3 :  0x0
C@1671ms: 	 2 : ShInitialize 0x102b4de0
C@1671ms: 	 1 : RtlKnownExceptionFilter 0x77e85b30
C@1671ms: 	 0 : RtlInitializeExceptionChain 0x77e4989f

```

**Comment by MartinBriza** [comment]
@ckooiker Thanks for reaching back to me and confirming it's okay for you!
@kaleem4 The fact this happens at the time of an attempt to connect to a HTTP server suggests this may be related to #80. Please try running this and tell me if it helped: https://mbriza.fedorapeople.org/FedoraMediaWriter-win32-4.0.95-14-g4da2072.exe

**Comment by kaleem4** [comment]
Still Not Working :(

**Comment by MartinBriza** [comment]
@kaleem4 After reading more about your configuration, it seems you have one of the graphics card that don't support OpenGL acceleration and their DirectX support is so poor they can't even handle ANGLE. You'll probably have to wait until there's software rendering support built in Qt we have in Fedora (which will come in summer).

**Comment by alfuro** [comment]
Hello @MartinBriza:
     Forgive me for jumping in but the FMW 4.0.8 was instantly crashing on Windows 10 (Creators Update) immediately after launch. I downloaded and installed 4.0.95 and its working like a charm.

thanks

**Comment by bmuessig** [comment]
I have the same problem. On Windows 10, 32-bit version 4.0.95 crashes immediately on start.

**Comment by MartinBriza** [comment]
@bmuessig Hello, can you please try 4.1.0? There were some further changes done to avoid crashes. In case it still does crash, please take a look into your Documents folder, you should find a log file there. Please send it to me or upload it here.

**Comment by GetzMikalsen** [comment]
I had the same issue, 4.1.0 fixed this issue.
I'm running windows 10 insider preview.

**Comment by golmert** [comment]
I am having crushes on WIN 10 and writer does not load. From reading earlier posts it is unfortunately a very common issue which happens to many people and makes me wonder how stable is this version of media writer? Why is it recommended in the Fedora website if this does work for us?? I read so many people have issue with this. As a non Linux user I decided to give Fedora a try and now I am so frustrated with this,Just makes me think twice why move that direction...

**Comment by MartinBriza** [comment]
Should be fixed by 8f6c16d1adbcf15c4319953259e76d7e70c85951
